Lesson05

nllbr opened the pull request.

nllbr committed:
Delete .gitignore

nllbr committed:
Delete README.md

nllbr committed:
Delete allhw

nllbr committed:
Create task01

не смогла сделать через менеджер файлов

nllbr committed:
Create task02

nllbr committed:
Create task03

nllbr committed:
Create task04

почему-то не работает с третьим элементом. не понимаю почему.

nllbr committed:
Create task05

nllbr committed:
Create task06

ShapovaLOVALOVA commented on task02:
```diff
@@ -0,0 +1,8 @@
+with open("text_2", "r") as f_obj:
+    for line in f_obj:
+        count += 1
+print(f'количество строк равно {count}')
```
так, а количество слов в каждой строке?=)

ShapovaLOVALOVA commented on task05:
```diff
@@ -0,0 +1,15 @@
+    f_obj.write(numbers)
+    f_obj.seek(0)
+    text = f_obj.read()
+    text_copy = text[::2]
```
очень интересное решение, взять каждый второй элемент, зная, что каждое число будет разделено пробелом. Только будет ошибка, если число будет двузначным, трехзначным и т.д.

ShapovaLOVALOVA reviewed (commented):
Нелли, хорошие решения, не увидела решения задачи 6 и 7.

nllbr committed:
Update task02

посмотрела разбор в следующем уроке, но как посчитать кол-во слов в строке, если вот у меня в файле еще и тире. он его считает как слово потому что оно отделено пробелом

nllbr committed:
Update task01

nllbr committed:
Update task03

nllbr committed:
Update task06

сделала по вашему разбору, но почему-то показывает вот такую ошибку.
Traceback (most recent call last):
  File "/Users/nllbr/PycharmProjects/lession05/h6.py", line 14, in <module>
    lessons_count = [int(lessons.rstrip('(л)(пр)(лаб)')) for lesson in lessons if lesson != '—']
  File "/Users/nllbr/PycharmProjects/lession05/h6.py", line 14, in <listcomp>
    lessons_count = [int(lessons.rstrip('(л)(пр)(лаб)')) for lesson in lessons if lesson != '—']
AttributeError: 'list' object has no attribute 'rstrip'

nllbr committed:
Create task07

сделала также с помощью разбора в следующем уроке
